app/src/exporters/ffmpeg.ts

```diff
@@ -0,0 +1,73 @@
+import ffmpeg from 'fluent-ffmpeg/lib/fluent-ffmpeg';
+import fs from 'fs';
+import path from 'path';
+import os from 'os';
+import { Source } from '../core/document';
+import { RenderItem } from '../state/editor';
+
+function encodePartAsMp3(source_path: string, part: RenderItem, output: string) {
```

Comment:
I would actually prefer a single-pass approach here, but this is ok for now as it is much easier
(see https://video.stackexchange.com/questions/10396/how-to-concatenate-clips-from-the-same-video-with-ffmpeg for a possible alternative)

Reply:
maybe we could use something like https://github.com/tedconf/fessonia in combination with https://www.npmjs.com/package/ffmpeg-static